foundations/foundation-common/src/main/java/io/servicecomb/foundation/common/net/NetUtils.java

```diff
@@ -201,4 +209,33 @@ public static boolean canTcpListen(InetAddress address, int port) {
       return false;
     }
   }
+
+  /**
+   * For security reason, 0.0.0.0 is not allowed to be bound to,
+   * and should be replaced with {@link #hostAddress}
+   *
+   * @param address address in configuration
+   * @return if host is 0.0.0.0, replace it with {@link #hostAddress};
+   * otherwise, return as it is.
+   */
+  private static String checkAddress(String address) {
+    if (!address.contains(COLON)) {
+      LOGGER.error("unexpected address format");
+      return null;
+    }
+    String ip = address.substring(0, address.indexOf(COLON));
+    InetAddress inetAddress = null;
+    try {
+      inetAddress = InetAddress.getByName(ip);
+    } catch (UnknownHostException e) {
+      LOGGER.error("host {} is unknown.", ip);
+      return null;
+    }
+    if (inetAddress.isAnyLocalAddress()) {
```

Comment:
Is there any security reason for this change?
We need to tell add the release-note to tell user they cannot use "0.0.0.0" any more.

Reply:
I misunderstood the requirement. I've communicated with liubao and jimin, and ensure that our current mechanism is right. This pull request will be closed later.